serpentTools/parsers/bumat.py

```diff
@@ -13,11 +12,11 @@ class BumatReader(MaterialReader):
         path to the depletion file
     """
 
-    def __init__(self, 
+    def __init__(self):
+        raise NotImplementedError
 
     if materialname==None or materialfile==None:
```

Comment:
This whole block should really be put into either 

- a separate function that is outside of the class, or
- worked into the class structure as the class `bumat.read` method

Having the __init__ raise this `NotImplementedError` defeats the purpose of using the class.

Reply:
Right, I intend to change this once the contributing guide gets set up, and once I make this work with how you all store data in a Material object.